YcmdCompletion.sublime-settings

```diff
@@ -46,4 +46,19 @@
     and other tools, uncomment this:
   */
   // "python_binary_path": "/usr/local/bin/python"
+
+  /*
+    The languages you wish to use ycmd for.
+    Supported: cpp, rust
+
+    The following languages may also work, but are untested with this plugin:
+    c, erlang, java, d, lua, python,
+    ocaml, js, perl, go, objc,
+    cpp, cs, php, ruby, rust
+
+    These languages may also work if the corresponding Sublime
+    syntax plugin is installed:
+    vb, typescript, elixir
+  */
```

Comment:
I'd better disable ability of adding non-supported languages. So I propose:
1) Add here a list of supported languages (and maybe some link to file with LANG_MAP definition)
2) When loading this activated language list, use only those, that are in LANG_MAP. For each other option print error to console.